tests/regions/raw_sensor_test.py

```diff
@@ -0,0 +1,91 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import json
+import os
+import shutil
+import tempfile
+import unittest
+
+from nupic.engine import Network
+from htmresearch.support.register_regions import registerAllResearchRegions
+
+class RawSensorTest(unittest.TestCase):
+  """ Super simple test of RawSensor """
+
+  @classmethod
+  def setUpClass(cls):
+    cls.tmpDir = tempfile.mkdtemp()
+
+  @classmethod
+  def tearDownClass(cls):
+    if os.path.exists(cls.tmpDir):
+      shutil.rmtree(cls.tmpDir)
+
```

Comment:
Because those are executed once for each call to a test method. setUpClass is only executed once for the whole suite. Granted, this file only has one method, but we might add more.